code/_core/obj/item/weapon/ranged/spellgem/armor_other.dm

```diff
@@ -0,0 +1,22 @@
+/obj/item/weapon/ranged/spellgem/projectile_armor
+	name = "armor other spell gem"
```

Comment:
"reinforce other" sounds like a better spell name

Reply:
I like it.